Sources/Core/Microsoft.StreamProcessing/Operators/Shufflecast/ShufflecastPipe.cs

```diff
@@ -165,7 +165,7 @@ public unsafe void OnNext(StreamMessage<TInnerKey, TSource> batch)
             }
         }
 
-        private void AddPunctuationOrLowWatermarkToAllBatches(StreamMessage<TInnerKey, TSource>[] batches, long syncTime, long otherTime, TInnerKey innerKey, int hash)
```

Comment:
>long otherTime [](start = 122, length = 14)

This looks like a bug - we should be setting vother = otherTime

Reply:
Understood - the real bug was that the parameter was not being used.